packages/components/src/Toast/Toast.css

```diff
@@ -5,20 +5,25 @@
 
 .container {
   display: flex;
-  flex-direction: column;
-  align-items: center;
   position: fixed;
+  right: 0;
   bottom: var(--space-large);
-  left: 50%;
+  left: 0;
   z-index: var(--elevation-toast);
+  width: 100%;
```

Comment:
You might think "should we cap this width on larger screens?" and I was thinking maybe, but the guidelines around `Toast` recommend short messages so I think it's fine to not cap this until we run into an issue

Reply:
^ The content of these toast messages could get created/pumped in from something that a developer isn't paying much attention to, or the content changes later and nobody notices that the toast got out of control. 
How much work would it be to have the Toast be a bit more "defensive" or robust in this case? It might be worth it to include now.